.github/workflows/integration.yaml

```diff
@@ -0,0 +1,52 @@
+name: Integration
+on: [push, pull_request]
+env: 
+  GALAXY_TEST_DBURI: 'postgres://postgres:postgres@localhost:5432/galaxy?client_encoding=utf8'
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [2.7]
+        subset: ['datatype']
```

Comment:
```suggestion
        subset: ['upload_datatype']
```

Reply:
Thanks, I've rebased this into the initial commit.